api-gateway/content/nginx.conf

```diff
@@ -69,6 +70,19 @@ http {
             proxy_read_timeout   3600s;
             proxy_redirect       off;
         }
+		
+        location /iothubmanager-svc/ {
+            rewrite              /iothubmanager-svc/(.*) /$1 break;
+            proxy_pass           $iothubmanager_svc_endpoint;
+            proxy_pass_header    Authorization;
+            # TODO ~devis: remove - https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/issues/11
+            # Public preview only: used to distinguish internal/external traffic
+            proxy_set_header     X-Source external;
+            proxy_buffering      off;
+            client_max_body_size 0;
```

Comment:
What type of requests are going to be sent to this service? Are we sure that none of these requests will contain body content?